docs/intro.md

```diff
@@ -8,23 +8,26 @@ description: Intro to Rollkit, a modular framework for rollups.
 
 Welcome to Rollkit docs. We’re happy you made it here!
 
-Our mission is to empower developers to quickly innovate and create entire new classes of rollups with minimal tradeoffs.
```

Comment:
maybe worth reworking this into the page somewhere?

Reply:
we're using it in "What's Rollkit" towards the end of the section